src/Mod/TechDraw/App/DrawProjectSplit.cpp

```diff
@@ -514,25 +426,49 @@ std::vector<TopoDS_Edge> DrawProjectSplit::scrubEdges(std::vector<TopoDS_Edge>&
 //        Base::Console().Message("DPS::scrubEdges(2) - origEdges is empty\n");     //debug
         return std::vector<TopoDS_Edge>();
     }
-    //HLR usually delivers overlapping edges. We need to refine edge overlaps
-    //into non-overlapping pieces
-    std::vector<TopoDS_Edge> noOverlaps;
-    noOverlaps = DrawProjectSplit::removeOverlapEdges(origEdges);
 
-    //HLR algo does not provide all edge intersections.
-    //need to split edges at intersection points.
-    std::vector<TopoDS_Edge> splitEdges;
-    splitEdges = DrawProjectSplit::splitIntersectingEdges(noOverlaps);
+    TopTools_ListOfShape edgeList;
```

Comment:
Would TopTools_IndexedMapOfShape (unique edges) be better than TopTools_ListOfShape (duplicate edges) here?  IDK, just asking.